src/main/java/com/example/exercise1/controller/HelloWorldController.java

```diff
@@ -36,4 +42,14 @@ public long getRemainingTime() {
         return timeMilli;
     }
 
+    @PostMapping(
+            value = "/api/sayHi"
+//            consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE}
+    )
+    public Map<String, Object> sayHi(@RequestBody User body) {
+        Map<String, Object> map = new HashMap<String, Object>();
```

Comment:
Can we return an object for example HiResponse here?

Reply:
yes, sure

Reply:
Please also include the class `User ` in your next commit too